tools/stats/print_test_stats.py

```diff
@@ -700,6 +700,7 @@ def build_info() -> ReportMetaMeta:
         "build_branch": os.environ.get("CIRCLE_BRANCH", ""),
         "build_job": os.environ.get("JOB_BASE_NAME", ""),
         "build_workflow_id": os.environ.get("CIRCLE_WORKFLOW_ID", ""),
+        "build_start_time_epoch": int(os.path.getmtime(os.path.realpath(__file__))),
```

Comment:
clever! I didn't realize this was a thing we could do; however, I think your test plan should clarify what exactly this is supposed to measure and how we can confirm that it is indeed measuring that thing

Reply:
done. modified the description